warp/src/blink/mod.rs

```diff
@@ -47,6 +47,9 @@ pub trait Blink: Sync + Send + SingleHandle + DynClone {
     /// peers included in the Vec<DID>.
     async fn offer_call(
         &mut self,
+        // May want to associate a call with a RayGun conversation.
+        // This field is used for informational purposes only.
+        conversation_id: Option<Uuid>,
```

Comment:
if this is `Option::Some`, you might then want to confirm the conversation exist within raygun via `RayGun::get_conversation`, and probably subscribe to the events from raygun to detect if the conversation been deleted so it would drop/end the call (though this, again, is optional)